views/default/walled_garden.css

```diff
@@ -0,0 +1,58 @@
+body {
+	background-image: url('walled_garden.jpg');
```

Comment:
This only allows replacement with other JPGs. #8721 Not a blocking issue though.